ClientLib/src/main/java/com/o3dr/services/android/lib/drone/mission/MissionItemType.java

```diff
@@ -113,6 +114,18 @@ public MissionItem getNewItem() {
         }
     },
 
+    DO_LAND_START("Do Land start") {
```

Comment:
@daimonkor from reading the wiki, it seems `DO_LAND_START` is only implemented for planes.

In that case, I'd suggest adding a `isTypeSupported(Type vehicleType)` method to this `MissionItemType` enum class, with the default implementation returning `true`, and which is overrode by the `DO_LAND_START` enum to return `true` only when `vehicleType.getDroneType() == Type.TYPE_PLANE`.